src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -1,39 +1,37 @@
 package be.seeseemelk.mockbukkit.inventory;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
-
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Don't import the entirety of `java.util` ._.

You should try to avoid wildcard imports in general.